www/templates/html/CompactFeedList.tpl.php

```diff
@@ -28,9 +28,9 @@ class="dcf-h-8 dcf-w-8"
             </div>
         </div>
         <div class="mh-video-label dcf-txt-center">
-            <span class="title"><?php echo UNL_MediaHub::escape($channel->title); ?></span>
+            <a class="dcf-card-link title" href="<?php echo $feed_url ?>" title="<?php echo UNL_MediaHub::escape($channel->description); ?>"><?php echo UNL_MediaHub::escape($channel->title); ?></a>
```

Comment:
If title can be empty then this is fine otherwise might be best to remove title attribute since channel description is not required.

Reply:
Otherwise looks good.

Reply:
I agree. The link doesn't need the `title` attribute so I've removed it.